profiles/common/modules/features/cce_basic_config/cce_basic_config.info

```diff
@@ -42,15 +46,19 @@ dependencies[] = password_policy
 dependencies[] = print
 dependencies[] = print_mail
 dependencies[] = print_pdf
-dependencies[] = print_ui
+# The print_pdf_pdf_tool variable is set in cce_basic_config_strongarm_alter().
+# Please do not remove this entry when re-exporting the variables.
```

Comment:
This comment should be present in a README file too because I do not think people read comment in such file.